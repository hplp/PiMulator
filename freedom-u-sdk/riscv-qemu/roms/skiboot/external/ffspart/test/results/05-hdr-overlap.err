Adding partition 'THREE' would cause partition 'ONE' at 0x00000200 to overlap with the header
Couldn't add entry 'THREE' 0x00000400 for 0x00000100
