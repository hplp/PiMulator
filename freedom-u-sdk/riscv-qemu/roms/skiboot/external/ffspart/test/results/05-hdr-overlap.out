Adding 'ONE' 0x00000200, 0x00000100
Adding 'TWO' 0x00000300, 0x00000100
Adding 'THREE' 0x00000400, 0x00000100
Freeing hdr
